packages/react-component-library/src/components/DatePicker/DatePicker.tsx

```diff
@@ -335,6 +335,20 @@ export const DatePicker = ({
         <FocusTrap focusTrapOptions={focusTrapOptions}>
           <StyledDayPicker
             firstDayOfWeek={1}
+            todayButton="Jump to today"
+            onTodayButtonClick={(day) => {
+              if (isDateDisabled(day, disabledDays || [])) {
+                return
+              }
+
+              const newState = handleDayClick(day)
+              dispatch({ type: DATEPICKER_ACTION.REFRESH_HAS_ERROR })
+              dispatch({ type: DATEPICKER_ACTION.REFRESH_INPUT_VALUE })
+
+              if (newState.endDate || !isRange) {
+                setTimeout(() => close())
```

Comment:
I wonder if it should stay open when clicked.

Reply:
Maybe worth asking Ella and Dan what they think about this behaviour.

Reply:
Do you mean just for the today button or when any date is clicked? Currently it is closed when any other date is picked